### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by anand2312**
Hi I made mkdocs

I have added mkdocs basic config, and added a github action that will redeploy docs on push to master

pls approve

u can see what it looks like here: https://anand2312.tech/snakecord

### CODE REVIEW

**Comment by asleep-cult** [comment]
can `mkdocs.yml` be in docs folder or something?

**Comment by anand2312** [comment]
> 
> 
> can `mkdocs.yml` be in docs folder or something?

this is the format in which mkdocs makes it, idk how you'd make it work if you put the config file in there 🤔 
sus

**Comment by asleep-cult** [comment]
what is `.github/workflows/build_docs.yml` ? 😳

**Comment by anand2312** [comment]
> 
> 
> what is `.github/workflows/build_docs.yml` ? 😳

auto deploy docs to github pages on push to ~~main~~ master

**Comment by asleep-cult** [comment]
I kinda don't want it to do that

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+## Examples
+
+```python
+import snekcord
+
+TOKEN = ''
+
+client = snekcord.UserClient(TOKEN)
```
Comment (ToxicKidz) [inline docs/getting_started.md:17]: Hmm, this doesn't really exist

Comment (anand2312) [reply docs/getting_started.md:17]: i pulled it from the existing README

Comment (asleep-cult) [reply docs/getting_started.md:17]: hmm interesting

**Comment by asleep-cult** [comment]
merge is confilic😢

**Comment by NovaTohsaka** [comment]
sus

**Pull request merged**